tests/mol-audio_no_clipbegin/OPS/content_001.xhtml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<html xmlns="http://www.w3.org/1999/xhtml">
+	<head>
+        <meta charset="utf-8" />
+		<title>Automatic start at the beginning of an audio track</title>
```

Comment:
Inconsistent indentation again. (Should there be an xx-mol-template folder, like https://github.com/w3c/epub-tests/tree/main/tests/xx-epub-template, and if so can you make sure the template is tidy?)

Reply:
The problem with a template is that there are many factors that make the tests fairly different from one another (smil files, audio files, etc). Let us come back to this issue when we are done with the tests.